validator/coordinator/src/main/java/tech/pegasys/artemis/validator/coordinator/ValidatorCoordinator.java

```diff
@@ -283,9 +285,9 @@ private void createBlockIfNecessary(
               deposits);
 
       this.eventBus.post(new ProposedBlockEvent(newBlock));
-      STATUS_LOG.log(Level.DEBUG, "Local validator produced a new block");
+      LOG.debug("Local validator produced a new block");
     } catch (SlotProcessingException | EpochProcessingException | StateTransitionException e) {
-      STATUS_LOG.log(Level.ERROR, "Error during block creation " + e.toString());
+      LOG.error("Error during block creation " + e.toString());
```

Comment:
Should go to std out, this will cause the user to lose money.

Reply:
Done 👍